docs/en/sdk-manual/android.md

```diff
@@ -260,57 +260,57 @@ All user attributes will be stored in `user` object, and all custom and global a
+| country_code     | country/region code for this device                                                           | generate from `java.util.Local.getCountry()`<br>default is: "UNKNOWN"                                                                                                                                                                                                         | data analysis needs                                                                                  |
+| sdk_version      | clickstream sdk version                                                                       | generate from `BuildConfig.VERSION_NAME`                                                                                                                                                                                                                                      | data analysis needs                                                                                  |
+| sdk_name         | clickstream sdk name                                                                          | this will always be "aws-solution-clickstream-sdk"                                                                                                                                                                                                                            | data analysis needs                                                                                  |
+| app_version      | the app version name of user's app.                                                           | generate from `android.content.pm.PackageInfo.versionName`<br>default is: "UNKNOWN"                                                                                                                                                                                           | data analysis needs                                                                                  |
+| app_package_name | the app package name  of user's app.                                                          | generate from `android.content.pm.PackageInfo.packageName`<br>default is: "UNKNOWN"                                                                                                                                                                                           | data analysis needs                                                                                  |
+| app_title        | the app's display name                                                                        | generate from `android.content.pm.getApplicationLabel(appInfo)`                                                                                                                                                                                                               | data analysis needs                                                                                  |
```

Comment:
too many minor mistakes to be corrected in this table

Reply:
corrected.